GitCommands/Settings/AppSettings.cs

```diff
@@ -358,6 +358,7 @@ public static bool ShowResetAllChanges
         public static readonly StringSetting ConEmuTerminal = new StringSetting("ConEmuTerminal", DetailedSettingsPath, "bash");
         public static readonly StringSetting ConEmuFontSize = new StringSetting("ConEmuFontSize", DetailedSettingsPath, "12");
         public static readonly BoolNullableSetting ShowGpgInformation = new BoolNullableSetting("ShowGpgInformation", DetailedSettingsPath, false);
+        public static readonly BoolNullableSetting MaximizeCommitMessageColumn = new BoolNullableSetting("MaximizeCommitMessageColumn", DetailedSettingsPath, false);
```

Comment:
I would prefer to have this behavior without configuration.

Reply:
As far as I know, drewnoakes and vbjay prefer a fixed size message column.
I guess the decision will only be about the default value of this setting.
... waiting for more comments

Reply:
I could be convinced to have it as the default.

Before #5087 (will I ever forget that PR number?) many columns stretched to fill the grid's width, which was annoying because fixing it when it looked bad often meant adjusting many columns, only to have them look bad again if the window resized. This only stretches one column, which seems sensible to me. I favour removing config options where possible.

Reply:
Lol. I'm sure you never will.